addons/mail/models/mail_notification.py

```diff
@@ -102,3 +103,47 @@ def _notification_format(self):
             'failure_type': notif.failure_type,
             'partner_id': [notif.res_partner_id.id, notif.res_partner_id.display_name],
         } for notif in self]
+
+    @api.model
+    def _cron_notify_admins(self):
+        """
+            Cron activity : Posts notifications about failed mails/sms and other batch failures...
+            This cron was created to avoid adding notification code (failure counter, '_notify_admins()' method...) while mail stack processed.
+        """
+        # Get the last check date (related to the cron period == 1 day)
+        previous_date = fields.Datetime.now() - timedelta(days=1)
```

Comment:
can we not use the interval from the cron directly ? If someone modifies the cron interval, it won't be aligned anymore.

Reply:
Something like :

cron = self.env['ir.cron'].sudo().search([('id', '=', self.env.ref('mail.ir_cron_mail_notify_administrators').id)], limit=1)

... - relativedelta(**{cron.interval_type: cron.interval_number})

@tde-banana-odoo What do you think ?

Reply:
cron = self.env.ref('mail.ir_cron_mail_notify_administrators')
is enough to get the cron :) 
but yeah, that's the idea

Reply:
Done :+1: